src/TGameDetails.h

```diff
@@ -535,6 +543,24 @@ class TGameDetails
                  "Rivendell, Goblin-town, Mirkwood, Dol Guldur, and the Mines of Moria. The game is "
                  "completely at no cost to play and has been continually enhanced since its "
                  "inception in the fall of 1991.")},
+
+            {qsl("MorgenGrauen"),
+             qsl("mg.mud.de"),
+             4711,
+             false,
+             qsl("<a href='http://mg.mud.de/'>mg.mud.de</a>"),
+             qsl(":/icons/morgengrauen_480x120.png"),
+             /* English translation, provided by Game:
+ *           "Welcome to Morgengrauen, the largest German-language "
+ *           "MUD, which has been continuously developed since 1992."
+ *           "\n\n"
+ *           "Play now or never!"
+ */
+             qsl("Willkommen im Morgengrauen, dem größten deutschsprachigen "
```

Comment:
```suggestion
             qsl(":/icons/morgengrauen_480x120.png"),
             /* English translation, provided by Game:
              * "Welcome to Morgengrauen, the largest German-language "
              *  "MUD, which has been continuously developed since 1992."
              * "\n\n"
              * "Play now or never!"
              */
             qsl("Willkommen im Morgengrauen, dem größten deutschsprachigen "
```